webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/EndpointsDefinition.scala

```diff
@@ -339,46 +339,70 @@ object DirectiveApi       extends ApiModuleProvider[DirectiveApi]
+  final case object ListAcceptedNodes  extends NodeApi with GeneralApi with ZeroParam with StartsAtVersion2 with SortIndex  {
     val z              = implicitly[Line].value
     val description    = "List all accepted nodes with configurable details level"
     val (action, path) = GET / "nodes"
   }
-  final case object GetNodesStatus          extends NodeApi with GeneralApi with ZeroParam with StartsAtVersion13 with SortIndex {
+  final case object GetNodesStatus     extends NodeApi with GeneralApi with ZeroParam with StartsAtVersion13 with SortIndex {
     val z              = implicitly[Line].value
     val description    = "Get the status (pending, accepted, unknown) of the comma separated list of nodes given by `ids` parameter"
     val (action, path) = GET / "nodes" / "status"
   }
-  final case object ListPendingNodes        extends NodeApi with GeneralApi with ZeroParam with StartsAtVersion2 with SortIndex  {
+  final case object ListPendingNodes   extends NodeApi with GeneralApi with ZeroParam with StartsAtVersion2 with SortIndex  {
     val z              = implicitly[Line].value
     val description    = "List all pending nodes with configurable details level"
     val (action, path) = GET / "nodes" / "pending"
   }
-  final case object PendingNodeDetails      extends NodeApi with GeneralApi with OneParam with StartsAtVersion2 with SortIndex   {
+  final case object PendingNodeDetails extends NodeApi with GeneralApi with OneParam with StartsAtVersion2 with SortIndex   {
     val z              = implicitly[Line].value
     val description    = "Get information about the given pending node"
     val (action, path) = GET / "nodes" / "pending" / "{id}"
   }
-  final case object NodeDetails             extends NodeApi with GeneralApi with OneParam with StartsAtVersion2 with SortIndex   {
+  final case object NodeDetails        extends NodeApi with GeneralApi with OneParam with StartsAtVersion2 with SortIndex   {
     val z              = implicitly[Line].value
     val description    = "Get information about the given accepted node"
     val (action, path) = GET / "nodes" / "{id}"
   }
-  final case object NodeInheritedProperties extends NodeApi with GeneralApi with OneParam with StartsAtVersion11 with SortIndex  {
+
+  final case object NodeInheritedProperties extends NodeApi with GeneralApi with OneParam with StartsAtVersion11 with SortIndex {
     val z              = implicitly[Line].value
-    val description    = "Get all proporeties for that node, included inherited ones"
+    val description    = "Get all propreties for that node, included inherited ones"
     val (action, path) = GET / "nodes" / "{id}" / "inheritedProperties"
   }
-  final case object ApplyPolicyAllNodes     extends NodeApi with GeneralApi with ZeroParam with StartsAtVersion8 with SortIndex  {
+
+  final case object NodeGlobalScore extends NodeApi with GeneralApi with OneParam with StartsAtVersion11 with SortIndex {
```

Comment:
We need documentation for these two new API

Reply:
I thought we wanted them private for now, don't we ? 

Also version is not ok yet

Reply:
ah yes, private seems more secure for now.